patten_a06c.js

```diff
@@ -0,0 +1,52 @@
+/*Bryan Patten
+patten_a06c.js
+INFO 2124
+Lisa Thoendel
+01/23/2020*/
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+while (lottoPicks.length < 5) {
+        let p = getLottoNumber();
+        lottoPicks.push(p);
+}
+/*While the length of the lottoPicks array is less than 5, a random number will be generated by the getLottoNumber
+function, that number will be stored in the variable p, then added to the lottoPicks array*/
+for (let i = 0; i < lottoPicks.length; i++) {
+    switch(getLottoNumber()) {
+        case 22: winnings = winnings + 10;

```

Comment:
in each of these cases - you would want to output that you found a match - like in the sample output:
Matched 22!